easybuild/easyconfigs/k/KofamScan_data/KofamScan_data-20230401-gompi-2021b.eb

```diff
@@ -0,0 +1,50 @@
+easyblock = 'Binary'
+
+name = 'KofamScan_data'
+version = '20230401'
+
+homepage = 'https://www.genome.jp/tools/kofamkoala/'
+description = """KofamKOALA assigns K numbers to the user's sequence data by 
+HMMER/HMMSEARCH against KOfam (a customized HMM database of KEGG Orthologs 
+(KOs)).  K number assignments with scores above the predefined thresholds 
+for individual KOs are more reliable than other proposed assignments.  Such 
+high score assignments are highlighted with asterisks '*' in the output.  
+The K number assignments facilitate the interpretation of the annotation 
+results by linking the user's sequence data to the KEGG pathways and EC 
+numbers.  [data only]."""
+# software_license = 'MIT'
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+source_urls = [
+    'https://www.genome.jp/ftp/db/kofam/',  # add archives/YYYY-MM-DD/ if moved
+]
+sources = [
+    'ko_list.gz',
+    'profiles.tar.gz',
+]
+checksums = [
+    '099a62576b1754f31d3182e54d8ea310b61a2cacd3f6b3d5762be51ab0c711dd',  # ko_list.gz
+    'e017ead8fd52677d0a20558b9fd504876558e1723e77ec6be79aa72e6dee90a2',  # profiles.tar.gz
+]
+
+dependencies = [
+    ('KofamScan',  '1.3.0'),  # hard coded latest program version
```

Comment:
I would have expected the dependency to be the other way around? Maybe not?

Reply:
It seems to me that KofamScan is updating the database files monthly and rather not associate database versions with program versions. There are tens of versions of database files but only four program versions. I am imaging that people may download KofamScan_data more frequently than install KofamScan module. So it is better to specify the most updated KofamScan when doing KofamScan_data installation.